### DISCUSSION TITLE
unknown shorthand flag: 'g' in -g`

### DISCUSSION THREAD

**Discussion by GiveBalmain** [pimcore/pimcore#14857] [Q&A / Support]
Command: docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.2-latest composer create-project pimcore/demo my-project
Error:
unknown shorthand flag: 'g' in -g`
See 'docker run --help'.
I don't know how to repair this issue.

**Comment by podarcis** [comment]
You are referring to the installation documentation https://github.com/pimcore/skeleton#docker

The shown error suggests that you accidentally provided the `-g` option to the docker command instead of letting your shell expand the 
```shell
`id -g`
```
using command substitution. Short: The syntax of your command has been entered incorrectly. Double check (or copy&paste) and try again.
